Core/Sources/AblyExtensions/ARTRealtimeChannels+Extensions.swift

```diff
@@ -0,0 +1,14 @@
+import Ably
+
+private let channelNamePrefix = "tracking"
+
+extension ARTRealtimeChannels {
+    func getChannelFor(trackingId: String, options: ARTRealtimeChannelOptions? = nil) -> ARTRealtimeChannel {
+        let segments = [channelNamePrefix, trackingId]
+        let channelName = segments.joined(separator: ":")
```

Comment:
Maybe we could simplify this to `let channelName = "tracking:" + trackingId` or something more Swift idiomatic but equally straightforward? :thinking: It's much easier to understand and reason about :wink:

Reply:
I think this is quite debatable. I did your idea initially but I moved to my current implementation because this is more extensible with little cost to readability:
```swift
        // your suggestion would look like this:
        let channelName = "\(trackingChannelNamespace):\(trackingId)" // or
        let channelName = trackingChannelNamespace + ":" + trackingId

        // my new implementation:
        let segments = [trackingChannelNamespace, trackingId]
        let channelName = segments.joined(separator: ":")
``` 

If we were to add more segments to the channel ID, mine would be just 1 variable in the segments. It can also be forgotted, imagine someone doing `trackingChannelNamespace + ":" + capabilityScope + trackingId`, and forgetting the `:`. This avoids that, and is quite rigid/ explicit that its joining the segments with colons.

Reply:
I agree with all the benefits that your solution brings but the question is if we are going to use them? (are we going to change this code in the near future?) :thinking: In such cases, I like to go with the YAGNI rule and assume that we won't and additionally try to stick to KISS rule and make it as simple as possible :wink: But it all boils down to personal preference in this case so I'm happy to go with your solution if that's what you prefer :smile: